nebula-algorithm/README.md

```diff
@@ -1,75 +1,85 @@
-# 欢迎使用 nebula-algorithm
-
-nebula-algorithm 是一款基于 [GraphX](https://spark.apache.org/graphx/) 的 Spark 应用程序，提供了以下图计算算法：
-
-
- |           算法名          |中文说明|应用场景|
- |:------------------------:|:-----------:|:----:|
- |         PageRank         |  页面排序  | 网页排序、重点节点挖掘|
- |         Louvain          |  社区发现  | 社团挖掘、层次化聚类|
- |          KCore           |    K核    |社区发现、金融风控|
- |     LabelPropagation     |  标签传播  |资讯传播、广告推荐、社区发现|
- |    ConnectedComponent    |  联通分量  |社区发现、孤岛发现|
- |StronglyConnectedComponent| 强联通分量  |社区发现|
- |       ShortestPath       |  最短路径   |路径规划、网络规划|
- |       TriangleCount      | 三角形计数  |网络结构分析|
- |   BetweennessCentrality  | 介数中心性  |关键节点挖掘，节点影响力计算|
- |        DegreeStatic      |   度统计   |图结构分析|
- 
-使用 `nebula-algorithm`，可以通过提交 `Spark` 任务的形式使用完整的算法工具对 `Nebula Graph` 数据库中的数据执行图计算，也可以通过编程形式调用`lib`库下的算法针对DataFrame执行图计算。
-
-## 如何获取
- 1. 编译打包 Nebula-Algorithm
+# Welcome to Nebula-Algorithm
+
+<p align="center">
+  <br>English | <a href="README-CN.md">中文</a>
+</p>
+
+nebula-algorithm is a Spark Application based on [GraphX](https://spark.apache.org/graphx/) with below Algorithm provided for now:
```

Comment:
Lacking article: the below Algorithm.
And in Nebula docs we use "following" more often.